src/main/java/com/uw/controller/LoginController.java

```diff
@@ -0,0 +1,68 @@
+package com.uw.controller;
+
+import com.uw.dto.LoginDTO;
+import com.uw.model.User;
+import com.uw.service.AuthService;
+import com.uw.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v1/login")
+public class LoginController {
+
+    private final UserService userService;
+    @Autowired
+    public LoginController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping
+    public ResponseEntity<String> login(@RequestBody LoginDTO loginDTO){
+        if( loginDTO.getUsername() == null || loginDTO.getPassword() == null ){
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("You must send all the fields");
```

Comment:
Given this is an user access validation then would the result a http 401 instead of bad request. Also message should be consistent with what you are validating. I mean. something like "Please provide a valid user and pwd"